mapbox_tilesets/scripts/cli.py

```diff
@@ -629,6 +630,109 @@ def callback(m):
         raise errors.TilesetsError(resp.text)
 
 
+@cli.command("upload-raster-source")
+@click.argument("username", required=True, type=str)
+@click.argument("id", required=True, callback=validate_source_id, type=str)
+@click.argument("inputs", nargs=-1, required=True, type=click.File("r"))
+@click.option("--quiet", is_flag=True, help="Don't show progress bar")
+@click.option(
+    "--replace",
+    is_flag=True,
+    help="Replace the existing source with raster source file ",
+)
+@click.option("--token", "-t", required=False, type=str, help="Mapbox access token")
+@click.option("--indent", type=int, default=None, help="Indent for JSON output")
+@click.pass_context
+def upload_raster_source(
+    ctx, username, id, inputs, quiet, replace, token=None, indent=None
+):
+    """Create a new raster tileset source, or add data to an existing tileset source.
+    Optionally, replace an existing tileset source.
+
+    tilesets upload-source <username> <source_id> <path/to/source/data>
+    """
+    return _upload_raster_source(
+        ctx, username, id, inputs, quiet, replace, token, indent
+    )
+
+
+def _upload_raster_source(
+    ctx, username, id, inputs, quiet, replace, token=None, indent=None
+):
```

Comment:
In order to support multiple `inputs`, I think you would also need to support multiple `id` values (i.e. can't have more than one raster per source id, right?).  If that's the case, may want to only support one file upload at a time for raster.

Reply:
Nevermind!